Source/Scene/getClippingFunction.js

```diff
@@ -1,18 +1,16 @@
 define([
```

Comment:
I noticed some bugs in here thanks to Webstorm highlighting:

<img width="277" alt="bug" src="https://user-images.githubusercontent.com/915398/37369526-b90394e6-26df-11e8-92bf-14ff7ff6847a.PNG">

The `indexOf` check is definitely a problem.
The second one is cosmetic, but would still be nice to fix. The file has a few occurrences of these issues.